src/tests/ftest/rebuild/delete_objects.yaml

```diff
@@ -35,5 +35,5 @@ container:
       record_qty: 1
   debug: True
 rebuild:
-  rank: 4
+  rank: 3
```

Comment:
If the existing rank 4 is being skipped/cancelled due to an existing issue, maybe the right thing to do here is to add a variant to run with rank 3, not replace. I think this would help avoid confusion.

Reply:
since DAOS-6865 has merged there is no reason to use rank 3 now.